src/geometry/stress.cpp

```diff
@@ -362,8 +380,43 @@ Stress::calc_stress_xc()
         stress_xc_ += t;
     }
 
-    symmetrize_stress_tensor(ctx_.unit_cell().symmetry(), stress_xc_);
+    // compute the contributions of the stress tensor originating from non-local corrections
+    if (ctx_.full_potential()) {
+        potential_.xc_vdw_stress<false>(density_);
```

Comment:
There is no stress code for full-potential. Maybe the switch is for non-local core correction?

Reply:
the template parameter is the same than in
```
 template <bool add_pseudo_core__>
void
Potential::xc(Density const& density__)
```

Reply:
you probably just need to keep `potential_.xc_vdw_stress<true>(density_);`